static/js/vue-cdr-access/tests/unit/browseSearch.spec.js

```diff
@@ -1,88 +1,95 @@
-import { createLocalVue, shallowMount } from '@vue/test-utils';
-import VueRouter from 'vue-router';
+import { shallowMount, flushPromises } from '@vue/test-utils';
+import  { createRouter, createWebHistory } from 'vue-router';
 import browseSearch from '@/components/browseSearch.vue';
+import displayWrapper from "@/components/displayWrapper";
 
 const query = 'Test Collection';
-let wrapper;
-const localVue = createLocalVue();
-localVue.use(VueRouter);
-const router = new VueRouter({
-    routes: [
-        {
-            path: '/record/uuid1234',
-            name: 'displayRecords',
-            query: ''
-        }
-    ]
-});
+let wrapper, router;
 
 describe('browseSearch.vue', () => {
-    beforeEach(() => {
+    beforeEach(async () => {
+        router = createRouter({
+            history: createWebHistory(process.env.BASE_URL),
+            routes: [
+                {
+                    path: '/record/:uuid',
+                    name: 'displayRecords',
+                    component: displayWrapper
+                }
+            ]
+        });
         wrapper = shallowMount(browseSearch, {
-            localVue,
-            router,
-            propsData: {
+            global: {
+                plugins: [router]
+            },
+            props: {
                 objectType: 'Folder'
             }
         });
+
+        await router.push('/record/1234');
```

Comment:
Just curious why the router needs to start would this path pushed now but didn't before?

Reply:
Not sure, but it seems to need something in the route history to look at.